integrated_channels/integrated_channel/management/commands/backfill_course_end_dates.py

```diff
@@ -0,0 +1,63 @@
+"""
+Update all courses associated with canvas customer configs to show end dates
+"""
+
+from django.contrib import auth
+from django.apps import apps
+from django.core.management.base import BaseCommand, CommandError
+from integrated_channels.canvas.client import CanvasAPIClient
+
+from integrated_channels.integrated_channel.management.commands import IntegratedChannelCommandMixin
+
+User = auth.get_user_model()
+
+
+class Command(IntegratedChannelCommandMixin, BaseCommand):
+    """
+    Update content transmission items to have their respective catalog's uuid.
+    """
+
+    def add_arguments(self, parser):
+        """
+        Add required arguments to the parser.
+        """
+        parser.add_argument(
```

Comment:
I'm curious if we need a catalog user argument at all, are we making any API calls to the catalog service? Is there a security reason why we'd need this or another user for the job? 🤔